Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionCommandCocoa.mm

```diff
@@ -202,6 +203,69 @@ - (void)performWebExtensionCommandForKeyCommand:(id)sender
     return stringBuilder.toString();
 }
 
+String WebExtensionCommand::userVisibleShortcut() const
+{
+    auto flags = modifierFlags();
+    auto key = activationKey();
+
+    if (!flags || key.isEmpty())
+        return emptyString();
+
+#if PLATFORM(MAC)
+    NSKeyboardShortcut *keyboardShortcut = [NSKeyboardShortcut shortcutWithKeyEquivalent:key modifierMask:flags.toRaw()];
+    return keyboardShortcut.localizedDisplayName;
```

Comment:
Not saving much with two lines. Also should make sure it returns empty string just in case the SPI returns `nil`, so we honor our non-null result.
```suggestion
    return [NSKeyboardShortcut shortcutWithKeyEquivalent:key modifierMask:flags.toRaw()].localizedDisplayName ?: @"";
```